document/js-api/index.bs

```diff
@@ -732,7 +732,7 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 
 <pre class="idl">
 dictionary GlobalDescriptor {
-  required USVString value;
+  required DOMString value;
```

Comment:
This doesn't make a difference, because it's only compared against strings that don't contain U+FFFD.

It would probably be better as an enum, though.